plugins/Web/config.py

```diff
@@ -95,4 +95,12 @@ def configure(advanced):
     seconds the bot will wait for the site to respond, when using the 'fetch'
     command in this plugin. If 0, will use socket.defaulttimeout"""))
 
+conf.registerGlobalValue(Web, 'useOembedRegistry',
+    registry.Boolean(False, _("""Determines whether the bot will use the 
+    oembed.com providers registry.""")))
+
+conf.registerGlobalValue(Web, 'useOembedDiscovery',
+    registry.Boolean(False, _("""Determines whether the bot will use HTML
+    discovery to find oEmbed endpoints.""")))
```

Comment:
Name the config variables `plugins.Web.oembed.registry` and `plugins.Web.oembed.discovery`.

Could you also make them channel-specific, but not op-settable?